app/src/test/java/org/simple/clinic/bp/entry/BloodPressureValidationTest.kt

```diff
@@ -184,15 +167,14 @@ class BloodPressureValidationTest {
   private fun instantiateFixture(openAs: OpenAs) {
     val effectHandler = BloodPressureEntryEffectHandler(
         ui = ui,
-        userSession = userSession,
-        facilityRepository = facilityRepository,
         patientRepository = patientRepository,
         bloodPressureRepository = bloodPressureRepository,
         appointmentsRepository = appointmentRepository,
         userClock = testUserClock,
-        schedulersProvider = TrampolineSchedulersProvider(),
-        uuidGenerator = FakeUuidGenerator.fixed(UUID.fromString("abb04673-6ec0-4e1a-a4ad-5380e6f7e233"))
-    ).build()
+        schedulersProvider = TestSchedulersProvider.trampoline(),
+        uuidGenerator = FakeUuidGenerator.fixed(UUID.fromString("abb04673-6ec0-4e1a-a4ad-5380e6f7e233")),
+        currentUser = { user }
+    ) { facility }.build()
```

Comment:
Same comment as above, moving this inside constructor.

Reply:
Oh wait, I think the IDE made this change automatically and I ended up correcting it for one of the test classes and not the others. 😅